data/json/items/containers.json

```diff
@@ -2718,6 +2719,172 @@
     ],
     "qualities": [ [ "CONTAIN", 1 ] ]
   },
+  {
+    "id": "aquarium_medium_drying",
+    "type": "GENERIC",
+    "category": "container",
+    "name": { "str": "50 gallon aquarium (drying)", "str_pl": "50 gallon aquariums (drying)" },
+    "description": "A 50 gallon glass aquarium, used for safely storing small animals for use as pets or food.  This one has been freshly made and requires time for the silicone to dry.",
+    "weight": "45 kg",
+    "volume": "222 L",
+    "longest_side": "91 cm",
+    "price": 5000,
+    "price_postapoc": 750,
+    "bashing": 1,
+    "material": [ "glass" ],
+    "symbol": ")",
+    "color": "light_cyan",
+    "pocket_data": [
+      {
+        "pocket_type": "CONTAINER",
+        "rigid": true,
+        "watertight": false,
+        "open_container": true,
+        "max_contains_volume": "189 L",
+        "max_contains_weight": "300 kg"
+      }
+    ],
+    "qualities": [ [ "CONTAIN", 1 ] ],
+    "use_action": {
+      "target": "aquarium_medium",
+      "msg": "You tear off the tape from the silicone, looks like the aquarium is ready.",
+      "moves": 100,
+      "type": "delayed_transform",
+      "transform_age": 604800,
+      "not_ready_msg": "The silicone hasn't dried quite yet.",
```

Comment:
I think it's better to give amount of time until seams dry. It could get annoying if player forgets when they made the aquarium, and the number is arbitrary anyway.

Reply:
I am not aware of any way to do this. The only other item that I know has a similar transformation code is the soft adobe brick which does not reference this either way. I also wouldn't call the number arbitrary. In the conditions of 25 celsius and 50% air humidity, it should take 24 hours to dry and a week to achieve maximum adhesion. This number is a week.